src/containers/character/PDFExport.jsx

```diff
@@ -165,6 +166,96 @@ export default function PDFExport(props) {
     download(pdfBytes, fileName, 'application/pdf')
   }
 
+  async function fillFormDAC() {
+    const formUrl = CHARACTER_SHEET_PURIST_DAC_URL
+    const formPdfBytes = await fetch(formUrl).then((res) => res.arrayBuffer())
+
+    const pdfDoc = await PDFDocument.load(formPdfBytes)
+
+    const form = pdfDoc.getForm()
+
+	const fields = form.getFields()
+	fields.forEach(field => {
+		const type = field.constructor.name
+		const name = field.getName()
+		console.log(`${type}: ${name}`)
+	})
+
+    const formFieldKeysOfficialSheet = {
```

Comment:
Most of these are the same as from the AAC purist character sheet, with a few exceptions:

1.  There is no `Description` field in place of the character drawing area, so those fields do not currently appear.
2. Many of the fields from the AAC purist sheet have " 2" appended on them, but none of the ones from the DAC template have them.  This has been changed to make it work.
3. AC and unarmoured AC are calculated with `19 - value`, which should be the correct conversion
4. all the "THAC" values have been hard-coded to level 1 values, as was the Attack Bonus on the AAC export.

Reply:
hey @jestin, thanks for your contribution. I will review this weekend and get back to you

Reply:
@jestin hey there, could you send me a link to the ascending AC PDF that you used? I can handle hosting it

Reply:
The PDF I'm using is just the DAC version of the "Purist" character sheet from Necrotic Gnome: https://necroticgnome.com/products/old-school-essentials-purist-character-sheet.  I renamed it `DAC.pdf`, but otherwise didn't modify it.  The only substantial difference I see between this one and the one you already host (besides DAC, of course), is that there is no form field for `Description` (see comments above), so the character details don't get inserted into the character portrait section of the character sheet.

Let me know if you need anything else (or if you have a way to add the `Description` form field), and I'll do my best to help out.

Reply:
@jestin thanks, apologies for the delay, this has now been merged